.travis.yml

```diff
@@ -10,8 +10,6 @@ env:  # generate list with: $ tox -l | xargs -I ITEM echo "  - TOXENV="ITEM
   - TOXENV=py27-django14
```

Comment:
Please use the command mentioned in the comment above to generate an env value list matching `tox.ini`:
```
$ tox -l | xargs -I ITEM echo "  - TOXENV="ITEM
```